debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -1015,7 +1015,7 @@ public enum SnapshotMode implements EnumeratedValue {
         /**
          * Perform a snapshot of the schema but no data upon initial startup of a connector.
          *
-         * @deprecated to be removed in Debezium 3.0, replaced by {{@link #NO_DATA}}
+         * @deprecated to be removed in Debezium 3.0, replaced by {{@link #no_data}}
```

Comment:
The link is supposed to be to the enum `NO_DATA`, so I believe the capitalization here is accurate, although I generally use `{@link #NO_DATA}` rather than the double-brackets, but IDEA seems to support both.